assets/src/scss/base/_icons.scss

```diff
@@ -8,51 +8,52 @@
   transition: fill .3s;
 }
 
-a.standalone-link {
+// This rule is applied to all link with its class names ends with "-link"
+a[class$="-link"]:not(.is-link):not(.nav-link) {
```

Comment:
I'm guessing this is why the PDF buttons look strange now, since they have the `.pdf-link` class 🤔